src/scripts/browserDeviceProfile.js

```diff
@@ -253,12 +254,12 @@ function supportsHlg(options) {
 
 function supportsDolbyVision(options) {
     return options.supportsDolbyVision ?? (false // eslint-disable-line sonarjs/no-redundant-boolean
-            || browser.safari && ((browser.iOS && browser.iOSVersion >= 13) || browser.osx)
+            || browser.safari && ((browser.iOS && browser.iOSVersion >= 13) || browser.osx || browser.vidaa)
```

Comment:
I don't think Vidaa is Safari.
```suggestion
            || browser.safari && ((browser.iOS && browser.iOSVersion >= 13) || browser.osx)
            || browser.vidaa
```